artemis-protocols/artemis-stomp-protocol/src/main/java/org/apache/activemq/artemis/core/protocol/stomp/v11/StompFrameHandlerV11.java

```diff
@@ -16,6 +16,7 @@
  */
 package org.apache.activemq.artemis.core.protocol.stomp.v11;
 
+import javax.security.cert.X509Certificate;
```

Comment:
This was implemented on core via https://issues.apache.org/jira/browse/ARTEMIS-261. This PR is just for Stomp. I believe other protocols will need to be examined on an individual basis.